src/vs/workbench/contrib/issue/electron-browser/issueService.ts

```diff
@@ -15,6 +15,7 @@ import { IWorkbenchIssueService } from 'vs/workbench/contrib/issue/electron-brow
 import { IWorkbenchEnvironmentService } from 'vs/workbench/services/environment/common/environmentService';
 import { INativeWorkbenchEnvironmentService } from 'vs/workbench/services/environment/electron-browser/environmentService';
 import { ExtensionType } from 'vs/platform/extensions/common/extensions';
+import { platform } from 'process';
```

Comment:
Why not `src/vs/base/common/platform.ts`?

Reply:
This matches the strings that were already used in the process explorer